conf/example/redis.json

```diff
@@ -0,0 +1,14 @@
+{
+  "region": "na-siliconvalley",
```

Comment:
Can you add a link in the README to the TCloud API docs, so we can see what fields are available

Reply:
don't think json allow us for link inside. The beauty of viper is it allow me to switch to yaml. Thoughts?